src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/external/UserSearchController.kt

```diff
@@ -24,6 +30,56 @@ class UserSearchController(
   private val userSearchService: UserSearchService
 ) {
 
+  @GetMapping("/search")
+  @Operation(
```

Comment:
Should the endpoint be /users/search - or if it's specific to external users then /externalusers/search ?

Reply:
This is declared at the top of the class: `@RequestMapping("/externalusers")`

Reply:
ah - I missed that - cheers